scripts/take_pic.sh

```diff
@@ -1,14 +1,45 @@
-#!/bin/bash
+#!/bin/bash -e
 
-P=$1
-T=$2
-F=$3
+usage() {
+  echo -n "##################################################
+# Take a picture via gphoto2.
+# 
+# This will change the camera setting to bulb and then take
+# an exposure for the requested amount of time.
+##################################################
+ $ $(basename $0) PORT EXPTIME OUTFILE
+ 
+ Options:
+  PORT              USB port as reported by gphoto2 --auto-detect, e.g. usb:001,004.
+  EXPTIME           Exposure time in seconds, should be greater than 1 second.
+  OUTFILE           Output filename (with .cr2 extension).
+
+ Example:
+  scripts/take_pic.sh usb:001,005 5 /var/panoptes/images/temp.cr2
+"
+}
+
+if [ $# -eq 0 ]; then
+    usage
+    exit 1
+fi
+
+PORT=$1
+EXPTIME=$2
+FILENAME=$3
 echo 'Taking picture'
-echo "T = ${T}s"
+echo "PORT = ${PORT}"
+echo "TIME = ${TIME}s"
+echo "FILE = ${FILENAME}"
 
 # Open shutter
-gphoto2 --port=${P} --set-config eosremoterelease=Immediate \
-         --wait-event=${T}s --set-config eosremoterelease=4 --wait-event-and-download=2s \
-         --filename "${F}"
+gphoto2 --port=${PORT} \
+        --set-config shutterspeed=0 \
```

Comment:
Could you add comments at the end of the lines where we have these magic numbers and symbols?

Reply:
Do you have a better syntax than this?

```bash
# Open shutter
gphoto2 --port=${PORT} \
        --set-config shutterspeed=0 `#Always set to bulb` \
        --set-config capturetarget=0 `#Capture to RAM for download` \
        --set-config eosremoterelease=Immediate \
```